clusters/overlays/yukon-cars2/patches/pullsecret-sealed-patch.yaml

```diff
@@ -0,0 +1,15 @@
1fKiJI+Dcca1Lk/b7xClfCwPkz8kwm01Z5Is+Q3jwQB3YwbdpQmxb0PuMNWYQ0rnrzzU0usyRjnh250C73+ezKPsCw+frLfNzLKslYrWCBlpVTFGZZUccDBrkfcV9v0OhiFpZVuPNfJDKpIYuK9Ocg+xvTxtcThfKbVMp3mdDLFwt5eHESrDku50YJjtr1j2Ps98UOd3/UHPisG6KhnTpReubAmTU0OgX63zTMUtnxvTNMX/5wjpMXKVhyXeztzAGnOHPOlyJ4cUsmjUi1CSw0d70nedChKeBJLhuaS55U28nvaUDuvTWImLCZZup/W0TiUQkSzo9KODQYZqBQ5OsLkveA8ckmGFC0rrlcBPJSMvXPBn4P6YrrkVFcczws5N6dgPiZsRApxaOAlKk/5GY6m6ivh2Hv41FUsiHMBGZCkYizf8ASkXWNjoEzEO4dzEs2ZOju8as6efKZXrlyT9GxEEStgW+oFIdofkwWBRIdEeAh6Bs6irxFDU8YVnNguq/ZySKWDGDhCQxXDEqcYDzjmtr31xaDGDGvp+unzYpyWxXOjKCnvTPTlpZLq4EjDsszXVPLqykE/EV448SGrcHWnXwO8bHGi4BIx+eVEYMJKqU7m2cdO5TkCoAOPECNmsVHB842zPkjeilVBIexh6woIMLyzb0P9SW7Pz8ibaIi/R/Jst9MzaEHQO3+dcQ48NuTyUqoww2F5u3xgYP6GL0NtIzb9BrGGX1Nd9qsEmy1mQc7NbTavhsMHecevxJi1RQgC+IQPUghu3SBdBJFbi47f93u+tmAKZt2u7oSSlvLdTPnE1iOuTeOmXUFck8UhVb3aQiQkQkpOQmf05U3qiKVGBbIu9FVgQxT/4z59F9rKXEspwrH+NXfyEU3sH6sSWPZyYlm1TDlzZ9R5dEEEOlnsFPlolKPawvGZ5oqOzq3dAArxoEo7+7GLOSQBd3S9lAcJYWkFMpI5TrhiQ2h4VzmD4KUTqhbMsyzA8DsqtYSjFcNpzJIv36FLAGrjBJWTSf7LNkAb+bCgtywdo/M+GAQ3iqPoYYKiC6RXlhVDmizCGm7U1/aSmb2H551hZmdn0pj5lu7I2ouMoLgRlDki5v22xibUpw7q8OtypTQ0tbr1vHEq7H1ZmpRPDZfjYCYCJ+9LHu3s416+/px7EKClHT7vnaKPWVmTMeeogMw6nMREfuw6euSrhiYN/QNST+roHklzFN8z0N+5WkRVo57Yj8hrvwuMkR3ujrfqTKNFMo1WX//xzrTjNmRie4diCx/V6zmEbFd8IoTLcAOEJjBIftIy8qayQk0iAh5JfCa0tWUz9EK+lJT1Cn2o3E33UwUSlVvzTCF21pQLpDXq+gi2szQ3JYy6R/fxPX21LG5xR3gZR5/tR/gruA7mAc6bEV5cMFfAYmm+iZmoAofneOakY2yzgrB8CtDYtj1oc3kcp6zkR9PpSmxcun7eOYeQTV4izvEe3H3jRehsEhf4+4YHN9HwX8VTnwLiQojNP+/bPWkM75A3nScPsV17TC0ARMJiaO+fdJbHn8wJw2crlZ8LQkRbd31g3XSZ6PWaH5eZGTTzP67uV3bYc1Q760YS18x/SSXCJ1unFyP6PwpeLLEzLc3PiuZMkjKzchU7vGMjYvlFTCOUsUTCY741mX74FoHevlR+azsv0DOKLrTWR2xtNNem6/VfQ0EyI9XcgCPJlqif0PyibXBnShmTxQbUnOz5Wk3wFade/AAG3nrNRXwQfbFTTVpwGS/VeQrL9QEUUs1SeNwF823VUBE9G0Us2adNNFg38Hz7qC9BqeAeE2vQkA5GRIZ1zsmHCbKaHCS3GlKzZqORozHMY4QAffeJjhAtqf/WyW3IFpd1Fo3emAMZSPtbEGSrvjz4DQE1hdtkNGGy/g1zDPljjeAPPLDtbEEam7SAbWk3bseKCmxCYMNfMACFoUsPd/w+QHSH2YgHQvZQwpdzHClcdCexBRAajOaXo67ijTampUz+18HNYn/2OHjYP03BJppw+aCW/716wRoKGP372y+b5m2yspUsl/5SN6zhZ6q+Iuzm6GNfqpiN04/AWDPFo6ZWCdspqNX4+f3P9ZyTeGy8nMGCCrNPRmJF/odmoWlPNtLc2rOH2KpMi39tQTv1r5ojTH3IeDFPFxZgMQUwEyyFxPYPzDdl1AZUqgP3lJxUDssnnzbpkNGguo9fzTziBWWrqkXhZrB0Du/ISP60zxLxyqzqWYENqByRQDfwLZg7JJI9gLfmIb87etfabRvG4j/dVZeaGsuWr7xpZSOAwa92FOiRZVn+KBAFyV0lSiTXo0nQRtWKDRxZFQGf3XMOXSczQ3xm8T/Eon84jAX3N7LQORb7+oEu3kd/yPI0yhE9Z7XWSfFmSm9tBaji2bocO3AgeBalEN46DIDwGGn6fwlS0bPYgngupECeg+iPJM7K3FHCMx6wVaLbHZaVC3P9SZ9zjEu5w+kZXp2FGM61qSHyiLHyP0H4963PKHyp2VCMFJZLuotN+BocPa+0oESY6UCVK5vWnzUJOIKmv5rS16mlY1P7x3/mXvVJ0B9F5TYW7WbXI9XNfXouoK/08II4odgyWFcdvBigF30VLh7Kza3am9wzpg+uTM0WTGNtOHcVikrUmagAtqJM+o9eDtDW1EIprn/FXX8vIV7bJ1sZgTWvNElIRS3Bnmb0+2ddFs8w9tyjKOufC/B48lIFep15XPyCz/0alGhJFpHEtaJerhIR7XSBbe3/KabOf75UNgLCnCe5mOvFuREuVHfPhPtdKWX1ApTOlzmzhZUw5RrgDJaRNgrWosLJMlgDFJgZF1fpUG9j6GRn0JJltB1gJPewDU5Hfh8rFN+MlCYQwZXQSNsiJVusTljcq7r72bAcfUiWiHMfcCf28xaPkjhbKoPKFCyqQMmXZE49GAjPre4Bv6sn6oOWgb94QUF5fpEWeD7j4yaVW78P4gapTY4VmW/2lxmX72sVxPqGCQzlB6zgKKOeyfQ='
```

Comment:
Is this your sealed pull secret? if this is something you created in another cluster you'll need to import the sealed secret key to this cluster or re-encrypt it

Reply:
yep 100%, something I just added to our vault, thanks for the reminder.